driver.go

```diff
@@ -1023,6 +1072,35 @@ func (d *cephRBDVolumeDriver) sh_unlockImage(pool, imagename, locker string) err
 	return nil
 }
 
+func (d *cephRBDVolumeDriver) sh_takeOverLock(pool, imagename string, locker Lock) error {
```

Comment:
The operation sequence has problem.  
Assume node1 is marked abnormal by HA. node1 has ongoing io1. 
Then node2 takes over the service and rm lock1.
Just before adding node1 to blacklist, node2 is crashed.
Then node3 takes over the service. as there is no lock exist, node3 adds lock2 and starts io2.  

io1/io2 may crrupt the rbd image.